tests/it/io/json/write.rs

```diff
@@ -231,9 +280,9 @@ fn write_list_of_struct() {
 
     assert_eq!(
         String::from_utf8(buf).unwrap(),
-        r#"{"c1":[{"c11":1,"c12":{"c121":"e"}},{"c11":null,"c12":{"c121":"f"}}],"c2":1}
+        r#"{"c1":[{"c11":1,"c12":null},{"c11":null,"c12":{"c121":"f"}}],"c2":1}
 {"c1":null,"c2":2}
-{"c1":[{"c11":5,"c12":{"c121":"g"}}],"c2":3}
+{"c1":[null],"c2":3}
```

Comment:
@jorgecarleitao Let me know if this test-case (list of structs) behavior is what was expected.

Reply:
Yap, makes a lot of sense to me: the item is null and thus the whole thing is just `null`.